modules/csg/editor/csg_gizmos.h

```diff
@@ -62,10 +65,48 @@ class CSGShape3DGizmoPlugin : public EditorNode3DGizmoPlugin {
 	~CSGShape3DGizmoPlugin();
 };
 
+class CSGShapeEditor : public Control {
+	friend class EditorPluginCSG;
```

Comment:
What's this for? You could just make the necessary members public.
Also this should be after GDCLASS.

Reply:
removed